tGUI.m

```diff
@@ -1,33 +1,33 @@
 %
-% tGUI(EXPERIMENT_NAME_STRING)
+% tGUI;
 %
-% Opens a Thermoesthesia GUI (tGUI) with figure title EXPERIMENT_NAME_STRING.
+% Waits for the arduino to read thermoneutral before it opens a 
+% Thermoesthesia GUI (tGUI) with figure title EXPERIMENT_NAME_STRING.
 %
 % If EXPERIMENT_NAME_STRING is not provided or is empty, it is set to TEST
 %
 % Starts a CSV file with name EXPERIMENT_NAME_STRING_YYYYMMDD_HHMMSS.csv
 % in the ./recordings/ folder
 %
-% CSV file has three columns:
+% CSV file has six columns:
 % 1. 's' : for seconds since GUI start (2 decimal places)
 % 2. 'level' : integer marker level (integers 0 to 100) ('very cold' to 'very hot')
 % 3. 'shiver' : shiver status (0=='none', 1=='shiver')
+% 4. 'green' : temperature read by green sensor
```

Comment:
Maybe we should change the sensor names from "Green" and "Blue" to the digital pin # on the Arduino. This will make the code more universal because not all users will label the sensors according to the color.